core/templates/dev/head/domain/statistics/PlaythroughImprovementTaskObjectFactorySpec.ts

```diff
@@ -174,28 +176,35 @@ describe('PlaythroughImprovementTaskObjectFactory', function() {
 
   beforeEach(angular.mock.inject(function(
       _$q_, _$rootScope_, _$uibModal_,
-      _PlaythroughImprovementTaskObjectFactory_,
-      _PlaythroughIssueObjectFactory_, _PlaythroughIssuesService_,
-      _PLAYTHROUGH_IMPROVEMENT_TASK_TYPE_) {
+      _PlaythroughImprovementTaskObjectFactory_, _PlaythroughIssuesService_,
+      _UserService_, _PLAYTHROUGH_IMPROVEMENT_TASK_TYPE_,
+      // TODO(#7222): Import these normally after
+      // PlaythroughImprovementTaskObjectFactory is upgraded to Angular 8.
+      ExplorationFeaturesService, PlaythroughIssueObjectFactory) {
     $q = _$q_;
     $rootScope = _$rootScope_;
     $uibModal = _$uibModal_;
     PlaythroughImprovementTaskObjectFactory =
       _PlaythroughImprovementTaskObjectFactory_;
-    playthroughIssueObjectFactory = _PlaythroughIssueObjectFactory_;
     PlaythroughIssuesService = _PlaythroughIssuesService_;
+    UserService = _UserService_;
     PLAYTHROUGH_IMPROVEMENT_TASK_TYPE = _PLAYTHROUGH_IMPROVEMENT_TASK_TYPE_;
 
-    PlaythroughIssuesService.initSession(expId, expVersion);
+    // TODO(#7222): Use these normally after
```

Comment:
Could you elaborate on what "Use these normally" means?

Reply:
Done.